### ISSUE TITLE
Misspelled first word after HTML element with absolute URL is not detected

### ISSUE THREAD

**Issue by rmartz** [Jason-Rev/vscode-spell-checker#201]
Using stock configuration in version 1.6.5, it appears that a misspelling in the first word after a `<a>` element that uses an absolute URL isn't detected. Relative URLs appear to work fine, though:

### Text:
```
<p>
    ghsjd <b>fhsdjkf</b> <i class='example'>fhsdjkf</i>
    <a href="http://example.org">fhdjskf fdshfjk</a>
    <a href="/example/">fhdskjf dfhjskd</a>
</p>
```

### Result:
<img width="415" alt="screen shot 2018-03-27 at 10 59 27 am" src="https://user-images.githubusercontent.com/1032849/37975908-4b90862e-31ae-11e8-839d-1be5d81c155c.png">

### Expected:
All words in the text flagged as misspelled, including `fhdjskf` on the third line.
